frontend/src/app/app.component.html

```diff
@@ -7,7 +7,7 @@
   </div>
 </nav>
 
-<div class="container">
+<div class="container py-4">
```

Comment:
`py-4` feels a bit much to me especially on the phone where there is not that much space. I would rather go with `py-3`.